packages/tse-backend/src/modules/exchange-data/model/exchange-data.model.ts

```diff
@@ -0,0 +1,130 @@
+import { AutoMap } from '@automapper/classes';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString, IsOptional, IsNumber } from 'class-validator';
+import { Transform } from 'class-transformer';
+
+export class GetTickersDto {
+  @AutoMap()
+  @ApiProperty({ example: 'binance' })
+  @IsString()
+  exchange: string;
+
+  @AutoMap()
+  @ApiProperty({ example: 'ETH/USDT,BTC/USDT' })
+  @IsString()
+  symbols: string;
+}
+
+export class GetOHLCVDto {
+  @AutoMap()
+  @ApiProperty({ example: 'binance' })
+  @IsString()
+  exchange: string;
+
+  @AutoMap()
+  @ApiProperty({ example: 'ETH/USDT' })
+  @IsString()
+  symbol: string;
+
+  @AutoMap()
+  @ApiProperty({ example: '1d', required: false })
+  @IsOptional()
+  @IsString()
+  timeframe?: string;
+
+  @AutoMap()
+  @ApiProperty({ example: '1609459200000', required: false })
+  @IsOptional()
+  @Transform(({ value }) => parseInt(value, 10))
+  @IsNumber()
+  since?: number;
+
+  @AutoMap()
+  @ApiProperty({ example: 30, required: false })
+  @IsOptional()
+  @Transform(({ value }) => parseInt(value, 10))
+  @IsNumber()
+  limit?: number;
+}
+
+export class GetTickerPriceDto {
+  @AutoMap()
+  @ApiProperty({ example: 'binance' })
+  @IsString()
+  exchange: string;
+
+  @AutoMap()
+  @ApiProperty({ example: 'ETH/USDT' })
+  @IsString()
+  symbol: string;
+}
+
+export class GetMultipleTickerPricesDto {
+  @AutoMap()
+  @ApiProperty({ example: 'binance,gate' })
+  @IsString()
+  exchangeNames: string;
+
+  @AutoMap()
+  @ApiProperty({ example: 'ETH/USDT,BTC/USDT' })
+  @IsString()
+  symbols: string;
+}
+
+export class GetSupportedSymbolsDto {
+  @AutoMap()
+  @ApiProperty({ example: 'binance' })
+  @IsString()
+  exchange: string;
+}
+
+export class GetTickersCommand {
+  @AutoMap()
+  exchange: string;
```

Comment:
Why isn't this an Enum?

Reply:
In our API, any exchange name can be passed, and if it's not configured (keys missing in env), the API returns a message; while ccxt lib could be used, we avoid this to keep modules separate and imports clean